wormhole/wormhole.go

```diff
@@ -33,10 +33,10 @@ type Client struct {
 	// DefaultRendezvousURL will be used.
 	RendezvousURL string
 
-	// TransitRelayAddress is the host:port address to offer
+	// TransitRelayURL is the proto://host:port address to offer
 	// to use for file transfers where direct connections are unavailable.
-	// If empty, DefaultTransitRelayAddress will be used.
-	TransitRelayAddress string
+	// If empty, DefaultTransitRelayURL will be used.
+	TransitRelayURL string
```

Comment:
We need to be careful given that this is a braking change. I don't know what plans of @psanford are with this but, if we were to release this in a minor release, would it be possible to do this with just deprecating the old field?